From 90c4bd0059367922f88768784bfb7c5ec1bc846c Mon Sep 17 00:00:00 2001
From: "Justin M. Forbes" <jforbes@fedoraproject.org>
Date: Thu, 3 Nov 2022 11:34:03 -0500
Subject: [PATCH] Revert "disable enum64 BTF in fedora rawhide"

This option is not supported by older pahole versions.

This reverts commit 7ab8e0d8b46b7f768e37f8ff39ccae2ecc897396.
---
 scripts/pahole-flags.sh | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/scripts/pahole-flags.sh b/scripts/pahole-flags.sh
index 81c8e082ec57..0d99ef17e4a5 100755
--- a/scripts/pahole-flags.sh
+++ b/scripts/pahole-flags.sh
@@ -20,7 +20,4 @@ if [ "${pahole_ver}" -ge "122" ]; then
 	extra_paholeopt="${extra_paholeopt} -j"
 fi

-# temporary workaround to disable enum64
-extra_paholeopt="${extra_paholeopt} --skip_encoding_btf_enum64"
-
 echo ${extra_paholeopt}
-- 
2.37.3

